AK/Math.h

```diff
@@ -244,7 +366,7 @@ constexpr T acos(T value)
     CONSTEXPR_STATE(acos, value);
 
     // FIXME: I am naive
-    return Pi<T> + asin(value);
+    return (T)M_PIl + asin(value);
```

Comment:
Dislike the cast. Can we really not use code similar to what we had before?

Reply:
I did not want to have the definitions in two places, but yes theoretically we could, and the c++ standard has these in a <numbers> header, we could adopt something similar, although maybe in a separate header.